WeakAuras/embeds.xml

```diff
@@ -17,7 +17,6 @@
   <Script file="Libs\LibDataBroker-1.1\LibDataBroker-1.1.lua"/>
   <Script file="Libs\LibButtonGlow-1.0\LibButtonGlow-1.0.lua"/>
   <Script file="Libs\LibSpellRange-1.0\LibSpellRange-1.0.lua"/>
-  <!-- TODO 8.0 <Script file="Libs\HereBeDragons\HereBeDragons-2.0.lua"/>-->
   <Script file="Libs\LibRangeCheck-2.0\LibRangeCheck-2.0.lua"/>
   <!--@end-no-lib-strip@-->
```

Comment:
while we are here, can you also remove this and the starting tag of it, we no longer support no-lib setups anyway and I forgot to remove that